package.json

```diff
@@ -288,11 +291,12 @@
     "markdown-it": "^8.4.0",
     "markdown-it-checkbox": "^1.1.0",
     "moment": "^2.22.1",
+    "telemetry-github": "https://github.com/shana/telemetry/releases/download/0.1.1/telemetry-github-v0.1.1.tgz",
```

Comment:
Running `vsce package

Reply:
my bad, I thought I had run `npm install` first but apparently hadn't 😱 This is fine.